enos/ci/enos-variables.hcl

```
@@ -0,0 +1,28 @@
+variable "terraform_plugin_cache_dir" {
+  description = "The directory to cache Terraform modules and providers"
+  type        = string
+  default     = null
+}
+
+variable "tfc_api_token" {
+  description = "The Terraform Cloud QTI Organization API token."
```

```suggestion
  description = "The Terraform Cloud QTI Organization API token"
```

This is to keep the use of periods consistent between var descriptions